app/services/image_upload.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+from flask_login import current_user
+from flask_uploads import IMAGES, UploadSet, configure_uploads
+from werkzeug.utils import secure_filename
+
+from app import app
+
+app.config["UPLOADED_PHOTOS_DEST"] = f"app/static/images/users/current_user"
+app.config["ALLOWED_IMAGE_EXTENSIONS"] = [".jpg", ".jpeg", ".png", ".gif"]
+photos = UploadSet("photos", IMAGES)
+configure_uploads(app, photos)
+
+
+def get_image(photo, name, url_path="users"):
+    app.config["UPLOADED_PHOTOS_DEST"] = f"app/static/images/{url_path}/{name}"

```

Comment:
It does not look like a good practice. I am not familiar with the Flask ecosystem, but I'm almost 100% sure that there is another (preferred) way to upload